docs/testing/ie-11-testing.md

```diff
@@ -1,12 +1,20 @@
 ---
 title: Internet Explorer 11 testing
 description: 'Test your Office Add-in on Internet Explorer 11.'
-ms.date: 05/19/2021
+ms.date: 06/18/2021
 localization_priority: Normal
 ---
 
 # Test your Office Add-in on Internet Explorer 11
 
+> [!IMPORTANT]
+> **Internet Explorer Still Used in Office Add-ins**
+>
+> Microsoft is ending support for Internet Explorer, but this doesn't significantly affect Office Add-ins. Some combinations of platforms and Office versions, including all one-time-purchase versions through Office 2019, will continue to use the webview control that comes with Internet Explorer 11 to host add-ins, as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). Moreover, support for these combinations, and hence for Internet Explorer, is still required for add-ins submitted to [AppSource](/office/dev/store/submit-to-appsource-via-partner-center). Two things *are* changing:
+>
+> - AppSource no longer tests add-ins in Office on the web using Internet Explorer as the browser. But AppSource still tests for combinations of platform and Office *desktop* versions that use Internet Explorer.
+> - The [Script Lab tool](../overview/explore-with-script-lab.md) will stop working in Internet Explorer sometime in 2021.
```

Comment:
You seem to have a couple of instances of this. Maybe use an include file?

Reply:
There are only 2 where it is exactly the same and it seemed to me that an include file was more trouble than it's worth when its only being used twice.